airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/DbtCloudTransformationsCard.tsx

```diff
@@ -52,7 +57,7 @@ export const DbtCloudTransformationsCard = ({ connection }: { connection: WebBac
       initialValues={{ jobs: dbtCloudJobs }}
```

Comment:
TODO on line 55 - not sure what the input focus issue is, but should that be taken care of in this PR?